frontend/src/components/graph/FolderView.vue

```diff
@@ -182,6 +207,44 @@ export default Vue.extend({
         name: targetRoute,
         params: { id }
       });
+    },
+
+    exportGraph(graph: DenseGraph) {
+      const FileDownload = require("js-file-download");
+      this.axios.get("graphs/" + graph.id + "/export").then(res => {
+        FileDownload(res.data, graph.name + ".xdsl");
+      });
+    },
+
+    importGraph(file: any) {
+      if (file != null) {
+        var formData = new FormData();
+        var imagefile = document.querySelector("#file");
```

Comment:
var -> const
und `document.querySelector("#file");` zu `this.$refs.NAME.$el` (musst dann oben beim element ref="NAME" machen)

imagefile ist whsl auch n falscher name :)

Reply:
das document.querySelector("#file"); ist unnötig. habe es weggemacht